ydb/library/yql/udfs/common/knn/knn-serializer.h

```diff
@@ -129,11 +143,17 @@ class TKnnBitVectorSerializer {
         }
     }
 
-    static const TArrayRef<const ui64> GetArray64(const TStringRef& str) {
-        const char* buf = str.Data();
-        const size_t len = (str.Size() - HeaderLen) / sizeof(ui64);
+    struct Array {
```

Comment:
According to style guide all the types should have a prefix 'T'
https://docs.yandex-team.ru/arcadia-cpp/cpp_style_guide
https://wiki.yandex-team.ru/kikimr/developers/Styleguide/

Reply:
Fixed